modules/db.ts

```diff
@@ -1,11 +1,12 @@
 import Dexie, { DBCore, DBCoreTable, DBCoreMutateRequest, Table } from 'dexie';
 
 export interface Item {
-  id: number;
+  id?: number;
```

Comment:
`id` and `isPurchased` are "defaulted" by the db (but it's not true defaulting, we're just injecting some middleware that assign these values on addition) so I had to make them optional here.

But they're not actually optional, I need to add a note to the project to look into handling this more eloquently.